main.tf

```
@@ -407,3 +407,41 @@ resource "github_repository_project" "repository_project" {
   name       = each.value.name
   body       = each.value.body
 }
+
+# ---------------------------------------------------------------------------------------------------------------------
+# Webhooks
+# ---------------------------------------------------------------------------------------------------------------------
+locals {
+  webhooks = [
+    for b in var.webhooks : merge({
+      name   = null
+      active = null
+      events = []
+      configuration = try(
+        merge({
+          url          = null
+          content_type = null
+          insecure_ssl = null
+          secret       = null
+        }, b.configuration)
+        , {}
+      )
+    }, b)
+  ]
+}
+
+resource "github_repository_webhook" "repository_webhook" {
+  count = length(local.webhooks)
```

missed that one suggestion to make it kind of complete.. ;)

```suggestion
  count = length(var.webhooks)
```